client/src/services/ApiClient.js

```diff
@@ -90,6 +90,23 @@ export async function addFundToWallet(clientID, increaseBy) {
   }
 }
 
+export async function createClient(client) {
+ 
+  const response = await fetch("http://localhost:3000/api/clients", {
+    method : 'POST',
+    headers: {'Content-Type': 'application/json'},
+    body: JSON.stringify({...client, fullName: client.firstName +" "+ client.lastName,
```

Comment:
check the object literal (stringify param). If it has properties not needed (like firstName, lastName after you have created fullName) just remove them. Maybe you can do this in handleSubmit after you are sure there aren't validation errors. In this way here you can just put client object as stringify param

Reply:
I couldn't do it as you say, can I leave it as it is. 😅 @G-Salviati